src/coprocessor/trie/mod.rs

```diff
@@ -591,7 +788,7 @@ impl<'a, F: LurkField, const ARITY: usize, const HEIGHT: usize> Trie<'a, F, ARIT
         let (arity_bits, bits_needed) = Self::path_bit_dimensions();
         let path = bits[bits.len() - bits_needed..]
             .chunks(arity_bits)
-            .map(|chunk| chunk.into())
+            .map(|chunk| chunk.iter().cloned().rev().collect())
```

Comment:
Cool, this lines up with the out-of circuit behavior. 

It would be great to have a rustdoc comment on endianness on `path` and `synthesize_path`. I've recently been bitten by such an issue (in a completely different context) and can vouch clear comments on the subject are a life-saver.